hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/ApproveAllowanceLogic.java

```diff
@@ -173,6 +178,7 @@ protected void applyNftAllowances(final List<NftAllowance> nftAllowances, final
             final var owner = allowance.getOwner();
             final var approvingAccount = fetchOwnerAccount(owner, payerAccount, accountStore, entitiesChanged);
             final var spenderId = Id.fromGrpcAccount(allowance.getSpender());
+            // To add allowances spender should be validated as being a valid account
```

Comment:
Following your pattern above, wouldn't we want to move this `loadAccountOrFailWith()` down before L#189,
```
  ...
                final var key = FcTokenAllowanceId.from(tokenId.asEntityNum(), spenderId.asEntityNum());
                if (allowance.getApprovedForAll().getValue()) {
                    // We skip this validation below when revoking an operator approval
                    accountStore.loadAccountOrFailWith(spenderId, INVALID_ALLOWANCE_SPENDER_ID);
                    approveForAllNfts.add(key);
                } else {
                    approveForAllNfts.remove(key);
                }
```

Reply:
Yes Michael, fixed. Added that on line 189 and also when `serialsCount > 0`. Thanks for pointing out 🙌